### DISCUSSION TITLE
I want to upload directory use @rpldy/upload-button.

### DISCUSSION THREAD

**Discussion by bigprongs** [rpldy/react-uploady#415] [Q&A] [answered]
I want to files upload in subdirectory use **@rpldy/upload-button**.
Is it possible?
If it is possible, how can I upload files use  **@rpldy/upload-button**?
I add reference link.
https://react-uploady-storybook.netlify.app/?path=/story/ui-uploady--with-exposed-internal-input

**Comment by yoavniran** [comment, marked as answer]
Hey @elitesuper 
check out this sandbox - https://codesandbox.io/s/react-uploady-webkitdirectory-rkl7mx?file=/src/App.js
Im working on a fix to allow passing `webkitdirectory` as boolean. Currently you have to pass it as string.

**Reply by bigprongs** [reply]
Thanks for your help!
